README.org

```diff
@@ -1,17 +1,46 @@
 #+TITLE: Ren'Py mode
 
-PyTom's old major mode for Ren'Py, previously maintained by treymerkley, now maintained by me.
+A major mode for editing Ren'Py files.
 
 ** Installation
 
   Either install within Emacs directly from MELPA or:
```

Comment:
Given that the package name doesn't follow the regular Emacs naming convention, the instructions to install from MELPA should probably specify the package name to use.